x/lockup/keeper/lock.go

```diff
@@ -219,6 +219,12 @@ func (k Keeper) GetLocksPastTimeDenom(ctx sdk.Context, denom string, timestamp t
 	return combineLocks(notUnlockings, unlockings)
 }
 
+func (k Keeper) GetLocksDenom(ctx sdk.Context, denom string) []types.PeriodLock {
+	unlockings := k.getLocksFromIterator(ctx, k.LockIteratorDenom(ctx, true, denom))
+	notUnlockings := k.getLocksFromIterator(ctx, k.LockIteratorDenom(ctx, false, denom))
+	return combineLocks(notUnlockings, unlockings)
+}
```

Comment:
pretty surprised we don't already have a function for this lying around, pretty sure we use it in distribution

Reply:
Hmmm... True... Did we / can we use `GetLocksLongerThanDurationDenom`?

Reply:
That worked!